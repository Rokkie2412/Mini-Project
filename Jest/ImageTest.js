import React from "react";
import { View,Image } from "react-native";

const ImageTest = () =>{
    return(
        <View>
            <Image source={require('./900628.png')}/>
        </View>
    )
}

export default ImageTest