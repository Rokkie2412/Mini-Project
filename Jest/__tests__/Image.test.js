import React from "react";
import { create,act } from "react-test-renderer";
import ImageTest from "../ImageTest";

const tree = create(<ImageTest/>)
test('Image Test',()=>{
    expect(tree).toMatchSnapshot()
})
