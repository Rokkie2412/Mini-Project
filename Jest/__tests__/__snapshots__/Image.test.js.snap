// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Test 1`] = `
<View>
  <Image
    source={1}
  />
</View>
`;
